test/get-current-committish.ts

```diff
@@ -0,0 +1,94 @@
+import test from 'ava';
+
+import './fixtures/globals';
+import {getCurrentCommittish} from '../source/github-helpers';
+
+// The titles supplied here listed here are real, not guessed, except the error tester
+test('getCurrentCommittish', t => {
+	// Error testing
+	t.is(getCurrentCommittish(
+		'/',
+		'some page title'
+	), undefined, 'It should never throw with valid input');
+	t.throws(() => getCurrentCommittish(
+		'https://github.com',
+		'github.com'
+	));
+
+	// Root
+	t.is(getCurrentCommittish(
+		'/typescript-eslint/typescript-eslint',
+		'typescript-eslint/typescript-eslint: Monorepo for all the tooling which enables ESLint to support TypeScript'
+	), undefined);
```

Comment:
This is the only case that needs to be changed by the way. You can fix this test by adding the extra element (the find link I think) on line 18 and remove it on line 27. To ensure it works correctly, also use a branch with slashes there

Reply:
How would I add an element here? (I have no idea how tests work)

Reply:
> How would I add an element here?

That's the line we were talking about in this PR:

```
document.head.insertAdjacentHTML('beforeend', '<a hidden="" data-hotkey="t" data-pjax="true" href="https://github.com/avajs/ava/find/master"></a>');
```

Instead of leaving it at the bottom of the page, move it to line 18 and then remove the element on line 27: document.querySelector('[data-hotkey="t"]')`